src/dom-manipulation.ts

```diff
@@ -8,85 +10,91 @@ import {
   convertMeterPerSecondToKilomentersPerHour,
 } from "./utils/utils";
 
-export function appendFlightInformationToFlightInfoContainer(
-  flight: IFlights
-): void {
-  const flightInfoDiv = document.getElementById("flights-info");
-  const flightButtonExistence = document.getElementById(
-    flight.icao24 + flight.callsign
-  );
-  if (!flightInfoDiv) return;
-  if (!flightButtonExistence) {
-    createNewFlightInfoRow(flight, flightInfoDiv);
-  } else {
-    appendExistingFlightInfoRow(flight);
-  }
-}
+// Sybscribe to api output and update table with new or updated info
+fetchStream$.subscribe((flightArray) => {
+  flightArray.map((flight) => {
+    const flightInfoDiv = document.getElementById("flights-info");
+    const flightButtonExistence = document.getElementById(
```

Comment:
Possible refactor of flightButtonExistence to flightButton.

Reply:
Makes sense, code adjusted as requested